src/GraphQL/Types/Composite/AutoRegisteringObjectGraphType.cs

```diff
@@ -225,26 +225,27 @@ protected virtual ArgumentInformation GetArgumentInformation<TParameterType>(Fie
         /// <summary>
         /// Returns a list of properties, methods or fields that should have fields created for them.
         /// <br/><br/>
-        /// Unless overridden, returns a list of public instance readable properties (including properties declared on
-        /// inherited classes) and public instance methods (excluding methods declared on inherited classes) that
+        /// Unless overridden, returns a list of public instance readable properties and public instance methods
+        /// (including properties and methods declared on inherited classes) that
         /// do not return <see langword="void"/> or <see cref="Task"/>.
```

Comment:
It's would be better to clarify tat part about void and Task corresponds only for methods.

Reply:
So should I change it? It would require a new PR, right?

```
/// Unless overridden, returns a list of public instance readable properties and public instance methods
/// that do not return <see langword="void"/> or <see cref="Task"/>
/// including properties and methods declared on inherited classes.
```

Reply:
I would change to stste this explicitly. Yes, small PR that I'll merge ASAP.